ui/yarn.lock

```diff
@@ -5921,6 +5921,9 @@ lines-and-columns@^1.1.6:
   resolved "https://registry.yarnpkg.com/lines-and-columns/-/lines-and-columns-1.1.6.tgz#1c00c743b433cd0a4e80758f7b64a57440d9ff00"
   integrity sha1-HADHQ7QzzQpOgHWPe2SldEDZ/wA=
 
+liquidjs@^9.23.3:
```

Comment:
I think we should avoid new deps if we can. Yarn is a security nightmare. Instead of generalizing this PR, can I ask you to just add specific variables for your use case?

Reply:
Got it. In this case, I plan to add:

- `status.startedAtEpoch`: Epoch seconds for `startedAt`
- `status.finishedAtEpoch`: Epoch seconds for `finishedAt`, if workflow/pod is still ongoing, use the current epoch timesamp

Reply:
Added those two variables. I've also updated the PR description to reflect the latest changes. Please take a look when available. Thanks!